### PR TITLE
Cleanup inline namespaces

### PR DESCRIPTION
**Pull request by lucteo**
Remove inline namespaces surrounding CPO types

### CODE REVIEW

**Comment by lucteo** [comment]
work in progress; currently we do this just for the set_* CPOs; we need it for all the CPOs

File: examples/algorithms/retry.hpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ struct _retry_sender {
   }
 
   template <class Env>
-  friend auto tag_invoke(stdex::get_sender_traits_t, const _retry_sender&, Env)
```
Comment (ericniebler) [inline examples/algorithms/retry.hpp:104]: The `get_sender_traits_t` type needs to be available. IIRC, LEWG asked for the tags to have name-able types.

Comment (lucteo) [reply examples/algorithms/retry.hpp:104]: All the tag types?
- `set_value_t`, `set_error_t`, `set_stopped_t`
- `connect_t`, `start_t`, `schedule_t`, ...
- `transfer`, `let_value`, `bulk`, ...

It means, I also need to change specification to show this.

Comment (ericniebler) [reply examples/algorithms/retry.hpp:104]: Yes

**Comment by lucteo** [comment]
Dropping this in the favour of #396

**Pull request closed without merging**